pgbot/routine.py

```diff
@@ -75,5 +75,5 @@ async def routine(guild: discord.Guild):
     routine functions to handle stuff
     """
     await handle_reminders(guild)
-    if random.randint(0, 3) == 0:
+    if random.randint(0, 2) == 0:
```

Comment:
This is probably not required, this makes snek get bored faster. Can you revert this line change?

Reply:
Okey, I'll fix that real quick ~~after pycharm finishes loading :pg_mike:~~